ui/app/templates/components/raft-storage-restore.hbs

```diff
@@ -0,0 +1,70 @@
+<PageHeader as |p|>
+  <p.top>
+    <nav class="key-value-header breadcrumb">
+      <ul>
+        <li>
+          <span class="sep">&#x0002f;</span>
+          {{#link-to "vault.cluster.storage"}}
+            Raft Storage
+          {{/link-to}}
+        </li>
+      </ul>
+    </nav>
+  </p.top>
+  <p.levelLeft>
+    <h1 class="title is-3">
+      Restore Snapshot
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+
+<form {{action (perform this.restore this.file) on="submit"}}>
+  <MessageError @errors={{this.errors}} />
+
+  {{#if this.isUploading}}
+    <div class="box is-sideless is-fullwidth is-marginless">
+    <AlertBanner
+      @type="warning"
+      @title="Uploading your file..."
+      @message="Raft snapshots can be very large files. Be prepared for uploading the file to take some time."
+      />
+    </div>
+    <div class="box is-fullwidth is-shadowless">
+    <button type="button" class="button" onclick={{action "cancelUpload"}}>
+      Cancel upload
+    </button>
+    </div>
+  {{else}}
+    <div class="box is-sideless is-fullwidth is-marginless">
+      <AlertBanner
+        @type="warning"
+        @title="This might take a while"
+        @message="Raft snapshots can be very large files. Be prepared for uploading the file to take some time."
+        />
+      <FileToArrayBuffer @onChange={{action (mut file)}} />
+      <div class="b-checkbox">
+        <input
+          type="checkbox"
+          id="force-restore"
+          class="styled"
+          checked={{this.forceRestore}}
+          onchange={{action
+            (mut this.forceRestore)
+            value="target.checked"
+          }}
+        />
+        <label for="force-restore" class="is-label">
+          Force restore
+        </label>
+        <p class="checkbox-help-text">
+        Bypasses checks ensuring the Autounseal or shamir keys are consistent with the snapshot data.
```

Comment:
should this be "bypass" instead of "bypasses"?

Reply:
I could go either way on this 😬 with "bypasses" being the description of what the option does. @joshuaogle what do you think?

Reply:
I [have been wording these](https://share.goabstract.com/2d96701e-9f36-444c-95ac-cadc6b6c25a2?sha=a6443254870a119ebd7888e2801df28b12924b88) like "[I want to] X", so in this case it would be:

```suggestion
        Bypass checks to ensure the AutoUnseal or Shamir keys are consistent with the snapshot data.
```